frontend/src/libs/helpers/get-url-with-query-string.helper.ts

```diff
@@ -0,0 +1,16 @@
+import { type AppQueryStringKey, type AppRoute } from '#libs/enums/enums.js';
+import { type ValueOf } from '#libs/types/types.js';
+
+type QueryParameters = Record<ValueOf<typeof AppQueryStringKey>, string>;
+
+const getUrlWithQueryString = (
```

Comment:
should accept path and parameters and return url, not function

Reply:
oh, I see. Ok, will rewrite it